actionview/lib/action_view.rb

```diff
@@ -86,6 +86,7 @@ module ActionView
 
   autoload :CacheExpiry
   autoload :TestCase
+  autoload :ERBTracker
```

Comment:
This isn't what we need yet. In my test app this puts `ERBTracker` at `ActionView::ERBTracker`. and we should keep it at `ActionView::DependencyTracker::ERBTracker`